README.md

```diff
@@ -35,6 +35,18 @@ end
 ```
 Learn more [here](https://github.com/chanzuckerberg/redis-memo/wiki/Auto-Invalidation-with-ActiveRecord).
 
+### Configure RedisMemo
```

Comment:
Let's make this an h2 (`##`) and move it right before the `Why RedisMemo?` section.